### ISSUE TITLE
[BUG] "No Artifacts Recorded Use the log artifact APIs to store file outputs from MLflow runs." 

### ISSUE THREAD

**Issue by Matinrokzz** [mlflow/mlflow#4400]
can anyone please help with this? i am using python and running on localhost

No Artifacts Recorded
Use the log artifact APIs to store file outputs from MLflow runs.

![image](https://user-images.githubusercontent.com/53460267/119797243-40086b00-bf04-11eb-9f0e-46c8b5eebca7.png)

**Comment by RMeli** [comment]
Is the variable `artifact_location` in your `meta.yml` file (within the `mlflow/*/` folder) correct?

I was experiencing the same problem; I run some experiments on machine and then additional experiments on a second machine using [MLFlow on localhost](https://mlflow.org/docs/latest/tracking.html#scenario-1-mlflow-on-localhost) and the `artifact_location` was pointing to the correct path for the first machine (and therefore failing with the error above when using the UI on the second machine).

**Comment by Matinrokzz** [comment]
yes, it's in my meta.yml, but I was running only 1 experiment at a time, maybe I am missing some part in my code.

**Comment by Euraxluo** [comment]
**
![image](https://user-images.githubusercontent.com/34028978/184270711-17d2fd0f-9367-49b4-a582-f05e10917b87.png)
**

**Comment by Euraxluo** [comment]
![image](https://user-images.githubusercontent.com/34028978/184270752-e451f1a6-37e3-4abb-8377-82d784e6b195.png)

**Comment by Euraxluo** [comment]
I have the same problem, can you help me

**Comment by Euraxluo** [comment]
![image](https://user-images.githubusercontent.com/34028978/184271313-986c4c0b-e05c-42ed-94b9-fb5e766f0cc6.png)

**Comment by Euraxluo** [comment]
![image](https://user-images.githubusercontent.com/34028978/184271544-898bfb6e-fe43-464b-93c3-354f6c7c47b9.png)

**Comment by Euraxluo** [comment]
![image](https://user-images.githubusercontent.com/34028978/184271638-ddfa4f18-fa0c-4a08-bf38-8a37eb799c94.png)

**Comment by Euraxluo** [comment]
> 



> Is the variable `artifact_location` in your `meta.yml` file (within the `mlflow/*/` folder) correct?
> 
> I was experiencing the same problem; I run some experiments on machine and then additional experiments on a second machine using [MLFlow on localhost](https://mlflow.org/docs/latest/tracking.html#scenario-1-mlflow-on-localhost) and the `artifact_location` was pointing to the correct path for the first machine (and therefore failing with the error above when using the UI on the second machine).

how to fix

**Comment by RMeli** [comment]
@Euraxluo IIRC, I just manually changed the variable to point to the correct location.
